packages/babel-plugin-dev/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import  PathModule from 'path';
+
+export default (api: any) => {
+  const {
+    types: t
+  } = api;
+
+  const isRootComponent = (path: any) => {
+    return path.parent.type !== "JSXElement" && path.node.openingElement.name.name !== "DevMenuWrapper";
+  };
+
+  const isScreenComponent = (state: any) => {
+    const screensDir = PathModule.join(state.opts.project, state.opts.screensDirectory);
+
+    if (!!state.opts.projectDevMenusDirectory) {
+      const projectDevMenusDir = PathModule.join(state.opts.project, state.opts.screensDirectory, state.opts.projectDevMenusDirectory);
+      return state.filename.includes(screensDir) && !state.filename.includes(projectDevMenusDir);
+    } else {
+      return state.filename.includes(screensDir)
+    }
+  }
+
+  return {
+    visitor: {
+      Program(path: any, state: any) {
+        if (isScreenComponent(state)) {
+          const identifier = t.identifier("DevMenuWrapper");
+          const importSpecifier = t.importSpecifier(identifier, identifier);
+          const importDeclaration = t.importDeclaration([importSpecifier], t.stringLiteral("fsdev"));
+
+          path.unshiftContainer("body", importDeclaration);
```

Comment:
this can be replaced with a babel import helper library maintained by babel: https://babeljs.io/docs/en/babel-helper-module-imports.html - note this will add a prefix `_` to prevent naming collisions....therefore when used must use `_<name>`